src/components/table/bl-table.ts

```diff
@@ -0,0 +1,285 @@
+import { CSSResultGroup, html, LitElement, PropertyValues, TemplateResult } from "lit";
+import { customElement, property, state } from "lit/decorators.js";
+import "element-internals-polyfill";
+import { event, EventDispatcher } from "../../utilities/event";
+import style from "./bl-table.css";
+import BlTableCell from "./table-cell/bl-table-cell";
+import BlTableHeaderCell from "./table-header-cell/bl-table-header-cell";
+import BlTableRow from "./table-row/bl-table-row";
+
+export const blTableTag = "bl-table";
+
+export const blSortChangeEventName = "bl-table-sort";
+export const blRowSelectChangeEventName = "bl-table-row-select";
+
+/**
+ * @tag bl-table
+ * @summary Baklava Table component
+ *
+ */
+@customElement(blTableTag)
+export default class BlTable extends LitElement {
+  static get styles(): CSSResultGroup {
+    return [style];
+  }
+
+  /**
+   * Selected table row selection key list
+   */
+  @property({ type: Array, reflect: true, attribute: "select-value" })
+  get selectValue(): string[] {
```

Comment:
The attribute name for select-value doesn't accurately reflect its purpose. I suggest renaming this property to align better, perhaps as selected-values or selected-rows.